game-core/src/main/java/games/strategy/engine/data/gameparser/XmlGameElementMapper.java

```diff
@@ -101,23 +101,10 @@ public XmlGameElementMapper(
         .put("TechActivationDelegate", TechActivationDelegate::new)
         .put("TechnologyDelegate", TechnologyDelegate::new)
         .put("UserActionDelegate", UserActionDelegate::new)
-        .putAll(newTwoIfBySeaDelegateFactories())
         .putAll(auxiliaryDelegateFactoriesByTypeName)
         .build();
   }
 
-  @SuppressWarnings(
-      "deprecation") // required for map compatibility; remove upon next map-incompatible release
```

Comment:
@RoiEXLab would you mind clarifying this comment? What exactly can be removed? The delegate 'TwoIfBySeaEndTurnDelegate" is used in `class_variations/../four_if_by_sea.xml`